app/src/main/java/com/conupods/OutdoorMaps/View/Directions/NavigationActivity.java

```diff
@@ -273,24 +257,19 @@ private void requestIndoorDirections() {
 
     // polyline function - can be modified to work with a single route
     private void addPolyLinesToMap(final DirectionsResult result) {
-        new Handler(Looper.getMainLooper()).post(new Runnable() {
-            @Override
-            public void run() {
-                for (DirectionsRoute route : result.routes) {
-                    List<com.google.maps.model.LatLng> decodedPath = PolylineEncoding.decode(route.overviewPolyline.getEncodedPath());
+        new Handler(Looper.getMainLooper()).post(() -> {
+            for (DirectionsRoute route : result.routes) {
```

Comment:
the logic to lay polylines needs to be in the run method

Reply:
thats the lambda function this should be fine